### ISSUE TITLE
EMOTIQ/CONFIG/GENERATE:NETWORK-GENERATE produces incorrect folder(s) names

### ISSUE THREAD

**Issue by echupriyanov** [emotiq/emotiq#307]
Given following input:
```lisp
(
  (
    :hostname "127.0.0.1"
    :ip "127.0.0.1"
    :gossip-server-port 65000
    :rest-server-port 3140
    :websocket-server-port 3145
  )
  (
    :hostname "127.0.0.1"
    :ip "127.0.0.1"
    :gossip-server-port 65001
    :rest-server-port 3141
    :websocket-server-port 3146
  )
  (
    :hostname "127.0.0.1"
    :ip "127.0.0.1"
    :gossip-server-port 65002
    :rest-server-port 3142
    :websocket-server-port 3147
  )
)
```
Calling `(emotiq/config:network/generate)` produces following:
```
2018-07-12 17:24:36 Writing configuration to '/var/tmp/conf/127.0.0.1-127.0.0.1-65002-3140-3145/'.
2018-07-12 17:24:36 Writing genesis block as JSON to /var/tmp/conf/127.0.0.1-127.0.0.1-65002-3140-3145/emotiq-genesis-block.json
2018-07-12 17:24:36 Writing genesis block as LOENC to /var/tmp/conf/127.0.0.1-127.0.0.1-65002-3140-3145/emotiq-genesis-block.loenc
2018-07-12 17:24:36 Writing configuration to '/var/tmp/conf/127.0.0.1-127.0.0.1-65002-3140-3145/'.
2018-07-12 17:24:36 Writing genesis block as JSON to /var/tmp/conf/127.0.0.1-127.0.0.1-65002-3140-3145/emotiq-genesis-block.json
2018-07-12 17:24:36 Writing genesis block as LOENC to /var/tmp/conf/127.0.0.1-127.0.0.1-65002-3140-3145/emotiq-genesis-block.loenc
2018-07-12 17:24:36 Writing configuration to '/var/tmp/conf/127.0.0.1-127.0.0.1-65002-3140-3145/'.
2018-07-12 17:24:36 Writing genesis block as JSON to /var/tmp/conf/127.0.0.1-127.0.0.1-65002-3140-3145/emotiq-genesis-block.json
2018-07-12 17:24:36 Writing genesis block as LOENC to /var/tmp/conf/127.0.0.1-127.0.0.1-65002-3140-3145/emotiq-genesis-block.loenc
```
But, actually, there should be 3 folders produced

**Comment by easye** [comment]
@echupriyanov Definitely my bug.  I will fix.

**Comment by echupriyanov** [comment]
Fixes were not merged into the `dev` branch. Reintroducing them with re-opened PR #309

**Comment by easye** [comment]
Should be fixed with #389

**Comment by easye** [comment]
Fixed with #389
